broker/src/main/java/org/apache/rocketmq/broker/processor/PullMessageProcessor.java

```diff
@@ -246,12 +246,6 @@ private RemotingCommand processRequest(final Channel channel, RemotingCommand re
             responseHeader.setMinOffset(getMessageResult.getMinOffset());
             responseHeader.setMaxOffset(getMessageResult.getMaxOffset());
 
-            if (getMessageResult.isSuggestPullingFromSlave()) {
```

Comment:
why it useless?

Reply:
This is the code used to relieve the pressure of the Master

Reply:
I think the following code covers this logic
`
            if (this.brokerController.getBrokerConfig().isSlaveReadEnable()) {
                // consume too slow ,redirect to another machine
                if (getMessageResult.isSuggestPullingFromSlave()) {
                    responseHeader.setSuggestWhichBrokerId(subscriptionGroupConfig.getWhichBrokerWhenConsumeSlowly());
                }
                // consume ok
                else {
                    responseHeader.setSuggestWhichBrokerId(subscriptionGroupConfig.getBrokerId());
                }
            } else {
                responseHeader.setSuggestWhichBrokerId(MixAll.MASTER_ID);
            }
`

Reply:
The logic is not the same

Reply:
@hzh0425 I believe it's fine to remove it 'cause they do have the same effect in the end, but welcome to provide your thoughts on this.